docs/template_plugin/tests/functional/op_reference/matrix_nms.cpp

```diff
@@ -461,11 +461,11 @@ std::vector<MatrixNmsParams> generateParams() {
 
 std::vector<MatrixNmsParams> generateCombinedParams() {
     const std::vector<std::vector<MatrixNmsParams>> generatedParams {
-        generateParams<element::Type_t::bf16, element::Type_t::f32, element::Type_t::i32>(),
-        generateParams<element::Type_t::f16, element::Type_t::f32, element::Type_t::i32>(),
+        generateParams<element::Type_t::bf16, element::Type_t::bf16, element::Type_t::i32>(),
```

Comment:
Should we keep old test cases and add new tests?

Reply:
The second type stands for the output type. Since the output type follows the input type and the old case will not  be correct and no needed any more.